integration/core_ext.yaml

```diff
@@ -0,0 +1,6 @@
+chaincode:
```

Comment:
before, `integration` directory contained only the tests but none of the scaffolding of the test.  I think it would help clarity if you could move scaffolding files into some subdirectory ...